front/pages/api/w/[wId]/labs/transcripts/index.ts

```diff
@@ -105,8 +104,8 @@ async function handler(
         await LabsTranscriptsConfigurationResource.makeNew({
```

Comment:
Do we support having two configurations per users?

Reply:
Not for now - I'd say let's wait to get the use case?